diff --git a/node_modules/ipld-explorer-components/dist/components/StartExploringPage.js b/node_modules/ipld-explorer-components/dist/components/StartExploringPage.js
index 5aeaf82..e08436c 100644
--- a/node_modules/ipld-explorer-components/dist/components/StartExploringPage.js
+++ b/node_modules/ipld-explorer-components/dist/components/StartExploringPage.js
@@ -11,8 +11,8 @@ var ExploreSuggestion = function ExploreSuggestion(_ref) {
       type = _ref.type;
 
   return React.createElement(
-    'a',
-    { className: 'flex items-center lh-copy pl3 pl0-l pv3 bb b--black-10 link focus-outline', href: '#/explore/' + cid },
+    'div',
+    { className: 'flex items-center lh-copy pl3 pl0-l pv3 bb b--black-10 link focus-outline' },
     React.createElement(
       'span',
       { className: 'flex items-center justify-center w3 h3 flex-none br-100 tc', style: { background: colorForNode(type) } },
diff --git a/node_modules/ipld-explorer-components/dist/components/object-info/ObjectInfo.js b/node_modules/ipld-explorer-components/dist/components/object-info/ObjectInfo.js
index 2872093..c6c1436 100644
--- a/node_modules/ipld-explorer-components/dist/components/object-info/ObjectInfo.js
+++ b/node_modules/ipld-explorer-components/dist/components/object-info/ObjectInfo.js
@@ -99,7 +99,7 @@ var ObjectInfo = function ObjectInfo(_ref2) {
         return x === data.type;
       }) ? React.createElement(
         'a',
-        { className: 'link avenir ml2 pa2 fw5 f6 blue', href: 'https://ipfs.io/ipfs/' + cid, target: '_external' },
+        { className: 'link avenir ml2 pa2 fw5 f6 blue', href: 'https://gateway.btfs.io/btfs/' + cid, target: '_external' },
         t('ObjectInfo.gatewayLink')
       ) : null
     ),
